packages/extension-commerce-bm-seo/src/utils/route-match-utils.test.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {matchPath} from './route-match-utils'
+
+describe('matchPath', () => {
+    const NullComponent = () => null
+    const routes = [
+        {path: '/', component: NullComponent},
+        {path: '/about', component: NullComponent},
+        {path: '/contact', component: NullComponent},
+        {path: '/products/*', component: NullComponent},
+        {path: '/products/:id', component: NullComponent},
+        {path: '*', component: NullComponent}
```

Comment:
What happens is path is undefined? I remember mentioning that undefined paths also get treated as wild cards (not sure if that is true), but can we test that?